### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Dunemaster**
This PR updates dependencies, including parent pom, thus enabling build on Java 11

Also update target maven distribution to maven 3.6.2

### CODE REVIEW

**Comment by Dunemaster** [comment]
@mosabua
@ifedorenko
will you have a look?

File: pom.xml [hunk 1]
```diff
@@ -13,23 +13,25 @@
...
     <groupId>io.takari</groupId>
     <artifactId>takari</artifactId>
-    <version>25</version>
+    <version>28</version>
   </parent>
 
   <groupId>io.takari.maven</groupId>
   <artifactId>concurrent-build-logger</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>${revision}</version>
```
Comment (ifedorenko) [inline pom.xml]: No, don't parametrize project version, artifacts deployed to repositories should have have literal version in their pom.

File: src/test/projects/basic/pom.xml [hunk 2]
```diff
@@ -8,8 +8,8 @@
   <packaging>pom</packaging>
 
   <properties>
-    <mavenVersion>3.3.3</mavenVersion>
-    <mavenPluginPluginVersion>3.4</mavenPluginPluginVersion>
+    <mavenVersion>3.6.2</mavenVersion>
```
Comment (ifedorenko) [inline src/test/projects/basic/pom.xml]: Does this mean maven `3.3.3` is not supported any more?

File: src/main/provisio/maven-distro.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <artifactSet to="ext" ref="runtime.classpath" />
   </artifactSet>
 
-  <fileSet to="/conf/logging">
+  <fileSet to="/">
```
Comment (ifedorenko) [inline src/main/provisio/maven-distro.xml:15]: Can you elaborate on this? Is this change of behaviour in provisio or you want to move the files inside the distro?

**Review by ifedorenko — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Dunemaster) [reply pom.xml]: Fixed

File: src/test/projects/basic/pom.xml (hunk 2 above)
Comment (Dunemaster) [reply src/test/projects/basic/pom.xml]: I have reverted the change

File: src/main/provisio/maven-distro.xml (hunk 3 above)
Comment (Dunemaster) [reply src/main/provisio/maven-distro.xml:15]: Setting "to" attribute value has no effecti on proviso 0.1.16  and results in wrong directory structure on provision 0.1.56

**Comment by Dunemaster** [comment]
@ifedorenko will you look at the corrections?

**Comment by jvanzyl** [comment]
I'll take a look as soon as I can. Sometime this week.

**Comment by Dunemaster** [comment]
@jvanzyl  did you have an opportunity to look at the PR?

**Comment by cstamas** [comment]
This one is fixed, on master by https://github.com/takari/concurrent-build-logger/pull/7

**Pull request closed without merging**